src/controllers/RecipeController.js

```diff
@@ -75,7 +81,21 @@ const addRecipe = asyncHandler(async (req, res) => {
 // @route  DELETE /api/recipes
 // @access private: Logged in user can only delete recipes under their name
 const deleteRecipe = asyncHandler(async (req, res) => {
-  // TODO: Backend delete button (Issue 35)
+  const { id } = req.params;
+  if (!id) {
+    res.status(400);
+    throw new Error('Recipe id missing from query parameters!');
+  }
+  if (!verify(req.headers.authorization)) {
+    res.status(400);
+    throw new Error(
+      'Unauthorized access detected! Only the appropriate user can delete this!'

```

Comment:
'Unauthorized access detected! Only the recipe owner can delete this'

Reply:
done